extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -49,6 +52,8 @@ public function getContext( Title $title, $skinName ) {
 					'sourcePointDetectionUrl' => $sourcePointDetectionUrl,
 					'sourcePointRecovery' => ARecoveryModule::isEnabled(),
 					'yavliUrl' => $yavliUrl,
+					'pageFairWebsiteCode' => $wg->AdDriverPageFairWebsiteCode,
```

Comment:
What kind of code is that? It's definitely not secret since we move it to JS. I know we did something similar previously with SP but now I'm starting to think it makes no sense ;) @fraszczakszymon what do you think?